tools/wqlat_example.txt

```diff
@@ -0,0 +1,418 @@
+
+wqname = mlx5_cmd_0000:18:00.1
+     usecs               : count     distribution
+         0 -> 1          : 0        |                                        |
+         2 -> 3          : 10       |***********************                 |
+         4 -> 7          : 17       |****************************************|
+         8 -> 15         : 3        |*******                                 |
+        16 -> 31         : 2        |****                                    |
+
+wqname = private
+     usecs               : count     distribution
+         0 -> 1          : 0        |                                        |
+         2 -> 3          : 0        |                                        |
+         4 -> 7          : 11       |****************************************|
+         8 -> 15         : 6        |*********************                   |
+
+wqname = kblockd
+     usecs               : count     distribution
+         0 -> 1          : 40       |****************************************|
+         2 -> 3          : 14       |**************                          |
+         4 -> 7          : 4        |****                                    |
+         8 -> 15         : 3        |***                                     |
+        16 -> 31         : 2        |**                                      |
+
+wqname = mlx5_cmd_0000:18:00.0
+     usecs               : count     distribution
+         0 -> 1          : 0        |                                        |
+         2 -> 3          : 3        |*****************                       |
+         4 -> 7          : 7        |****************************************|
+
+wqname = xfs-log/dm-4
+     usecs               : count     distribution
+         0 -> 1          : 0        |                                        |
+         2 -> 3          : 1        |****************************************|
+
+wqname = mlx5_fw_tracer
+     usecs               : count     distribution
+         0 -> 1          : 0        |                                        |
+         2 -> 3          : 0        |                                        |
+         4 -> 7          : 1        |****************************************|
+
+wqname = writeback
+     usecs               : count     distribution
+         0 -> 1          : 0        |                                        |
+         2 -> 3          : 0        |                                        |
+         4 -> 7          : 4        |****************************************|
+         8 -> 15         : 1        |**********                              |
+
+wqname = xfs-cil/dm-4
+     usecs               : count     distribution
+         0 -> 1          : 0        |                                        |
+         2 -> 3          : 0        |                                        |
+         4 -> 7          : 1        |****************************************|
```

Comment:
The above dump with 'wqname' is too long, you can cut short with some '......' in between.

Reply:
okay, will do in next commit